src/Command/FrontendCommands.php

```diff
@@ -167,6 +173,47 @@ public function frontendInstall()
             return $result;
         }
         $this->io()->success('Front-end dependencies installed.');
+        $result = $this->taskExec("drush en -y dkan_js_frontend")->dir("docroot")->run();
+        if ($result->getExitCode() != 0) {
+            $this->io()->error('Could not install front-end node module');
+            return $result;
+        }
+        
+        if ($opts['theme']) {
+            $this->installTheme($opts['theme']);
```

Comment:
```suggestion
            $this->installTheme();
```
Now that `FrontendCommands::installTheme()` doesn't take parameters, `$opts['theme']` can be removed as an argument.